base/shared/src/main/scala/scalaz/data/compose.scala

```diff
@@ -0,0 +1,185 @@
+package scalaz
+package data
```

Comment:
Someone else please tell me if this is `data` or `ct` or something else. I can't tell and don't really care to puzzle it out.

Reply:
`ct`

Reply:
Alrighty then.